app/javascript/src/lib/OWLanguageLinter.js

```diff
@@ -622,3 +623,17 @@ function findUndefinedSubroutines(content) {
     }
   }
 }
+
+function findTripleEquals(content) {
+  const stringRanges = findRangesOfStrings(content);
+
+  for (const match of matchAllOutsideRanges(stringRanges, content, /={3}/g)) {
+    const from = match.index
+    diagnostics.push({
+      from,
+      to: from + match[0].length,
+      severity: "warning",
+      message: "This isn't JavaScript! Use single equals for assignments, or double equals to comparisons."
```

Comment:
While it's funny enough, for a lot of people this won't mean anything. I vote we leave out the "This isn't javascript!" bit

Reply:
```suggestion
      message: "Use single equals for assignments, or double equals to comparisons. Triple equals don't exist here (but they do in JavaScript)."
```

Reply:
Sure, that will do

Reply:
Oh ha, the last one is better in my opinion